cineast-core/src/main/java/org/vitrivr/cineast/core/db/cottontaildb/CottontailEntityCreator.java

```diff
@@ -143,8 +141,8 @@ public boolean createSegmentEntity() {
       this.cottontail.client.create(entity);
 
       /* Create indexes. */
-      this.createIndexWithoutSchemaName(MediaSegmentDescriptor.ENTITY, MediaSegmentDescriptor.FIELDNAMES[0], HASH_UQ, txId);
-      this.createIndexWithoutSchemaName(MediaSegmentDescriptor.ENTITY, MediaSegmentDescriptor.FIELDNAMES[1], IndexType.HASH, txId);
+      this.createIndex(entityName, MediaSegmentDescriptor.FIELDNAMES[0], IndexType.BTREE_UQ, txId);
```

Comment:
Is `HASH` now simply `BTREE` or are there also implementation changes in Cottontail?

Reply:
This is a cottontail change, xodus comes with new index types. Maybe @ppanopticon can elaborate?

Reply:
This is simply a rename. 

What used to be called HASH and HASH_UQ were Tree based indexes all along (even for MapDB). I just took the opportunity to reflect that in the naming of the different enumerations.